packages/react/slot/src/Slot.tsx

```diff
@@ -11,30 +11,28 @@ interface SlotProps {
 
 const Slot = React.forwardRef<HTMLElement, SlotProps>((props, forwardedRef) => {
   const { children, ...slotProps } = props;
-  const childLength = React.Children.count(children);
+  const childArray = React.Children.toArray(children);
 
-  if (childLength === 1) {
+  if (childArray.some(isSlottable)) {
     return (
-      <SlotClone {...slotProps} ref={forwardedRef}>
-        {children}
-      </SlotClone>
-    );
-  }
-
-  return (
-    <>
-      {React.Children.map(children, (child) => {
-        if (React.isValidElement(child) && child.type === Slottable) {
-          return (
+      <>
+        {childArray.map((child) => {
+          return isSlottable(child) ? (
             <SlotClone {...slotProps} ref={forwardedRef}>
               {child.props.children}
             </SlotClone>
+          ) : (
+            child
           );
-        }
+        })}
+      </>
+    );
+  }
```

Comment:
So this case if for things like `Tooltip` where it will find a `<Slottable>` internally?

Reply:
Yep 👍  it will slot **their** children and leave our children as is.